lambda/MAS/appsettings.json

```
@@ -23,7 +23,11 @@
     "SecretKey": "",
     "ServiceURL": "",
     "BucketName": "",
-    "StaticURL": ""
+    "StaticURL": "",
+    "CloudFront": {
+      "Enabled": "",
```

Should we set this to false by default so that if you're running it locally, it doesn't try to invalidate the cache. Just make sure we add an Octo var for it
```suggestion
      "Enabled": false,
```